rust_src/src/character.rs

```diff
@@ -1,12 +1,46 @@
 //! Operations on characters.
 
+use libc::{c_uchar, ptrdiff_t};
+
 use remacs_macros::lisp_fn;
 use remacs_sys::EmacsInt;
 
 use lisp::LispObject;
 use lisp::defsubr;
 use multibyte::{make_char_multibyte, raw_byte_from_codepoint_safe};
 use multibyte::MAX_CHAR;
+use threads::ThreadState;
+
+/// True iff byte starts a character in a multibyte form.
+///
```

Comment:
Copied typo "iff" from upstream.

Reply:
'iff'' is not a typo. It means "if and only if". There is no need for that extra strong wording here though. 'If' is sufficient.

Reply:
Something else that I've learned :)